prospector.yml

```diff
@@ -25,6 +25,7 @@ pylint:
     max-line-length: 100
   disable:
     - logging-format-interpolation
+    - inconsistent-return-statements
```

Comment:
I don't like this but you may don't agree.

```python
def run_time(self):
        """Total command runtime in seconds."""
        if self.start_time is not None and self.end_time is not None:
            diff = self.end_time - self.start_time
            return diff.seconds
```

in that example, we are forced to put a `return None` as the last line of the method.

Reply:
In general, I think we should enforce consistent returns but it doesn't strictly have to be fixed in this PR. I removed this and ran it to see how many places we do this. There's around 35 functions with inconsistent returns.

Reply:
Yes, there aren't too many functions. I added disabled it globally because I think it could be different opinions on this, so we need to agree if we want those explicit `return` or not.

If we are going in that direction I can fix those 35 functions.